packages/@aws-cdk/aws-glue/lib/job.ts

```diff
@@ -0,0 +1,872 @@
+import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
+import * as events from '@aws-cdk/aws-events';
+import * as iam from '@aws-cdk/aws-iam';
+import * as cdk from '@aws-cdk/core';
+import * as constructs from 'constructs';
+import { IConnection } from './connection';
+import { CfnJob } from './glue.generated';
+import { ISecurityConfiguration } from './security-configuration';
+
+/**
+ * AWS Glue version determines the versions of Apache Spark and Python that are available to the job.
+ *
+ * @see https://docs.aws.amazon.com/glue/latest/dg/add-job.html.
+ *
+ * If you need to use a GlueVersion that doesn't exist as a static member, you
+ * can instantiate a `GlueVersion` object, e.g: `GlueVersion.of('1.5')`.
+ */
+export class GlueVersion {
+  /**
+   * Glue version using Spark 2.2.1 and Python 2.7
+   */
+  public static readonly V0_9 = new GlueVersion('0.9');
+
+  /**
+   * Glue version using Spark 2.4.3, Python 2.7 and Python 3.6
+   */
+  public static readonly V1_0 = new GlueVersion('1.0');
+
+  /**
+   * Glue version using Spark 2.4.3 and Python 3.7
+   */
+  public static readonly V2_0 = new GlueVersion('2.0');
+
+  /**
+   * Custom Glue version
+   * @param version custom version
+   */
+  public static of(version: string): GlueVersion {
+    return new GlueVersion(version);
+  }
+
+  /**
+   * The name of this GlueVersion, as expected by Job resource.
+   */
+  public readonly name: string;
+
+  private constructor(name: string) {
+    this.name = name;
+  }
+
+  /**
+   * The glue version name as expected by job resource.
+   */
+  public toString(): string {
```

Comment:
let's just use property access when we need to get the actual value instead of these `toString` methods

Reply:
will do!